.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -51,7 +55,6 @@ notify-slack:
     - if: $DDR != "true"
       when: never
     - !reference [ .on_deploy_a7 ]
-  image: registry.ddbuild.io/slack-notifier:sdm
   tags: ["arch:amd64"]
```

Comment:
This is already part of the template and can be removed

Reply:
I see that we also redefine it in the the `notify` job: https://github.com/DataDog/datadog-agent/blob/main/.gitlab/notify.yml#L19-L41
So maybe this is clearer for the CI to keep the tag, if that's ok for you

Reply:
Definitely not a blocking comment :)